pkg/envoy/admin/access/access.go

```diff
@@ -0,0 +1,7 @@
+package access
+
+import "github.com/kumahq/kuma/pkg/core/user"
+
+type ConfigDumpAccess interface {
+	ValidateGetConfigDump(user user.User) error
```

Comment:
nit but I'd prefer `View` over `Get`, to differentiate it if we introduce `Get` access to resources.